Yoast/wpseo-news
# Bump plugin-grunt-tasks

## Summary

Bump plugin-grunt-tasks to v1.6

This PR can be summarized in the following changelog entry:

* None needed.

## Relevant technical choices:

* No changes to `aliases.yaml` needed in this one.

## Test instructions

This PR can be tested by following these steps:

* Make sure `build` task remains working.
